Makefile

```diff
@@ -37,17 +34,13 @@ vendor:
 tag: all
 	git tag -s $(TAGVER) -a -m "$(TAGMSG)"
 
-lint:
-	$(GOPATH)/bin/golint $(PACKAGE_NAMES) | tee /tmp/autograph-golint.txt
-	test 0 -eq $$(grep -c -Pv 'stutters|suggestions' /tmp/autograph-golint.txt)
-
 # refs: https://github.com/mozilla-services/autograph/issues/247
 check-no-crypto11-in-signers:
 	test 0 -eq $(shell grep -Ri crypto11 signer/*/ | tee /tmp/autograph-crypto11-check.txt | wc -l)
 
 show-lints:
-	-cat /tmp/autograph-golint.txt /tmp/autograph-crypto11-check.txt /tmp/autograph-staticcheck.txt
```

Comment:
I would rather we simply get rid of the `show-lints` target and output the status of the linters directly to console as they are running.

For example, I think the `check-no-crypto11-in-signers` target could just use the exit code from `grep` and be rewrittten as:

```
# refs: https://github.com/mozilla-services/autograph/issues/247
check-no-crypto11-in-signers:
	! shell grep -Ri crypto11 signer/*/
```

Reply:
I can do both! I was removing golint because it's unsupported and its docs say to replace it with go vet and staticcheck (like we've almost done). I'll pop up another PR to do that work